docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -206,7 +206,7 @@ Use the platform default configuration for all settings that affect screenshots,
 
 ### Include window shadow and rounded corners
 
-Screenshots must include the native toolkit shadow and rounding. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
+Screenshots must include the native toolkit shadow and rounding. If the titlebar is drawn by your platform instead, the titlebar and its shadow must be included. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

Comment:
What do you think about this?

```suggestion
Screenshots must include the native toolkit (or system, if the platform renders titlebars) shadow and rounding. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

Reply:
I'd say it should still be clear that the titlebars gotta be included, too, unless an example screenshot is added clearly showing a server-side decoration being included.

Maybe "native toolkit (or system if the platform renders titlebars) titlebar, shadow and rounding." might suffice?

Reply:
Also as an extra idea: Wouldn't it be better to say "toolkit's" and "system's" in this sentence?

Reply:
Hmm... guess I'll apply it for now and re-review, see what razzeee thinks after the fact.